app/src/main/java/com/inveitix/android/clue/database/DBHelper.java

```diff
@@ -52,6 +61,10 @@ public void insertRoom(Room room) {
         database.insert(DBConstants.DB_TABLE_ROOMS, null, cv);
     }
 
+    /**
+     * Insert MuseumMap data to maps table
```

Comment:
Repeats what the name of the method says. Either elaborate or remove documentation.